docs/pushnotificationios.md

```diff
@@ -143,26 +138,25 @@ details is an object containing:
 static scheduleLocalNotification(details: ScheduleLocalNotificationDetails);
 ```
 
-Schedules the localNotification for future presentation.
+Schedules a local notification for future presentation.
 
 **Parameters:**
 
 | Name    | Type   | Required | Description |
 | ------- | ------ | -------- | ----------- |
 | details | object | Yes      | See below.  |
 
-details is an object containing:
+`details` is an object containing:
 
-- `fireDate` : The date and time when the system should deliver the notification.
 - `alertTitle` : The text displayed as the title of the notification alert.
 - `alertBody` : The message displayed in the notification alert.
-- `alertAction` : The "action" displayed beneath an actionable notification. Defaults to "view". Note that Apple no longer shows this in iOS 10 +
-- `soundName` : The sound played when the notification is fired (optional).
-- `isSilent` : If true, the notification will appear without sound (optional).
-- `category` : The category of this notification, required for actionable notifications (optional).
+- `fireDate` : When the notification will be fired. Schedule notifications using either `fireDate` or `fireIntervalSeconds`, with `fireDate` taking precedence.
+- `fireIntervalSeconds` : Seconds from now to display the notification.
 - `userInfo` : An object containing additional notification data (optional).
-- `applicationIconBadgeNumber` The number to display as the app's icon badge. Setting the number to 0 removes the icon badge (optional).
-- `repeatInterval` : The interval to repeat as a string. Possible values: `minute`, `hour`, `day`, `week`, `month`, `year` (optional).
```

Comment:
Same here with `alertAction` and `repeatInterval` -- as I understand the JS will still accept these parameters but the are deprecated, not yet removed?